test/unit/url-mapper-spec.js

```diff
@@ -152,11 +106,43 @@ describe('url mapper', function() {
       check(specific.url, specific);
     });
 
+    it('matches, if no trailing slash', function() {
+      const noTrailing = {
+        url: 'http://foo.com',
+        newUrl: 'newUrl'
+      };
+
+      set(noTrailing);
+      check('http://foo.com', noTrailing);
```

Comment:
I don't really like those tests.
For me a unit tests should always end with one expect. These `set` and `check` functions make it much more complicating to read tbh.